ipld/retriever.go

```diff
@@ -61,6 +62,7 @@ func (r *Retriever) Retrieve(ctx context.Context, dah *da.DataAvailabilityHeader
 			if byzErr := r.checkForByzantineError(ctx, dah, err); byzErr != nil {
 				return nil, byzErr
 			}
+			log.Warnw("not enough shares to reconstruct data square, requesting more...", "err", err)
```

Comment:
Why warn level?

Reply:
Because it not an error and because you need to warn node operator about unhappy case/code path. Usually there should not be any retries

Reply:
Okay if it's not a frequent issue, then fine with me. Ack